WebContent/AdminLogin.html

```diff
@@ -128,6 +181,52 @@ <h1>Welcome to Online Book Store</h1>
 		</tr>
 	</table>
 	</div>
-
+	<script type="text/javascript">
+
+		// Captcha Script
+		
+		function checkform(theform){

```

Comment:
check this commit and change accordingly: https://github.com/shashirajraja/onlinebookstore/pull/58/commits/65cabe5524ecc79cff553966fbdbffff22f7e4dc

Also format the file (Use ctrl+shift+f)

Reply:
check this also